{\rtf1\ansi\ansicpg1252\cocoartf1038\cocoasubrtf360
{\fonttbl\f0\fswiss\fcharset0 Helvetica;}
{\colortbl;\red255\green255\blue255;}
\margl1440\margr1440\vieww9000\viewh8400\viewkind0
\pard\tx720\tx1440\tx2160\tx2880\tx3600\tx4320\tx5040\tx5760\tx6480\tx7200\tx7920\tx8640\ql\qnatural\pardirnatural

\f0\fs24 \cf0 diff -bru wxWidgets.orig/samples/notebook/notebook.cpp wxWidgets/samples/notebook/notebook.cpp\
--- wxWidgets.orig/samples/notebook/notebook.cpp	2014-07-31 21:16:47.000000000 -0700\
+++ wxWidgets/samples/notebook/notebook.cpp	2014-07-31 21:07:53.000000000 -0700\
@@ -177,16 +177,16 @@\
     // Create and add some panels to the notebook\
 \
     wxPanel *panel = CreateRadioButtonsPage(parent);\
-    parent->AddPage( panel, RADIOBUTTONS_PAGE_NAME, false, GetIconIndex(parent) );\
+    parent->AddPage( panel, RADIOBUTTONS_PAGE_NAME, false/*, GetIconIndex(parent)*/ );\
 \
     panel = CreateVetoPage(parent);\
-    parent->AddPage( panel, VETO_PAGE_NAME, false, GetIconIndex(parent) );\
+    parent->AddPage( panel, VETO_PAGE_NAME, false/*, GetIconIndex(parent)*/ );\
 \
     panel = CreateBigButtonPage(parent);\
-    parent->AddPage( panel, MAXIMIZED_BUTTON_PAGE_NAME, false, GetIconIndex(parent) );\
+    parent->AddPage( panel, MAXIMIZED_BUTTON_PAGE_NAME, false/*, GetIconIndex(parent)*/ );\
 \
     panel = CreateInsertPage(parent);\
-    parent->InsertPage( 0, panel, I_WAS_INSERTED_PAGE_NAME, false, GetIconIndex(parent) );\
+    parent->InsertPage( 0, panel, I_WAS_INSERTED_PAGE_NAME, false/*, GetIconIndex(parent)*/ );\
 \
     parent->SetSelection(1);\
 \}\
@@ -618,6 +618,10 @@\
     else // no old book\
     \{\
         CreateInitialPages(m_bookCtrl);\
+		m_bookCtrl->SetPageImage( 0, 0 );\
+		m_bookCtrl->SetPageImage( 1, 1 );\
+		m_bookCtrl->SetPageImage( 2, 2 );\
+		m_bookCtrl->SetPageImage( 3, 3 );\
     \}\
 \
     m_sizerFrame->Insert(0, m_bookCtrl, wxSizerFlags(5).Expand().Border());\
diff -bru wxWidgets.orig/samples/notebook/notebook.h wxWidgets/samples/notebook/notebook.h\
--- wxWidgets.orig/samples/notebook/notebook.h	2014-07-31 21:16:47.000000000 -0700\
+++ wxWidgets/samples/notebook/notebook.h	2014-07-31 21:08:09.000000000 -0700\
@@ -186,10 +186,10 @@\
     to decide what type of page it is.\
 */\
 \
-#define I_WAS_INSERTED_PAGE_NAME        wxT("Inserted")\
-#define RADIOBUTTONS_PAGE_NAME          wxT("Radiobuttons")\
-#define VETO_PAGE_NAME                  wxT("Veto")\
-#define MAXIMIZED_BUTTON_PAGE_NAME      wxT("Maximized button")\
+#define I_WAS_INSERTED_PAGE_NAME        wxT("N0")\
+#define RADIOBUTTONS_PAGE_NAME          wxT("N1")\
+#define VETO_PAGE_NAME                  wxT("N2")\
+#define MAXIMIZED_BUTTON_PAGE_NAME      wxT("N3")\
 \
 // Pages that can be added by the user\
 #define INSERTED_PAGE_NAME              wxT("Inserted ")\
}